roles/natnet.role

```diff
@@ -9,7 +9,8 @@ nuclear_role(
   support::configuration::GlobalConfig
   network::NUClearNet
   network::NetworkForwarder
-  platform::cm740::HardwareIO
+  platform::cm740::HardwareSimulator
```

Comment:
This doesn't seem like a change we would want?

Reply:
I've got to ignore this locally, using the simulator is easier for testing but I'll change it back